test/sql-luatest/defaults_test.lua

```diff
@@ -245,3 +245,21 @@ g.test_new_defaults = function()
         box.space.T:drop()
     end)
 end
+
+g.test_new_func_defaults = function()
+    g.server:exec(function()
+        -- Make sure an expression works correctly as a default value.
+        local sql = [[CREATE TABLE T(I INT PRIMARY KEY, A ANY DEFAULT(9 * 7));]]
```

Comment:
Can you please point for me a test which ensures that `DEFAULT (constant)` is equivalent to `DEFAULT constant`?

Reply:
I believe that would be lines 240-241 and 244 in the file:
```Lua
        local sql = [[CREATE TABLE T(I INT PRIMARY KEY, A ANY DEFAULT 123,
                      B STRING DEFAULT('something'));]]
        ...
        t.assert_equals(box.space.T:format()[3].default, 'something')
```